client/my-sites/woocommerce/index.js

```diff
@@ -1,17 +1,23 @@
+import page from 'page';
+import { createElement } from 'react';
 import { makeLayout } from 'calypso/controller';
+import { getSiteFragment } from 'calypso/lib/route';
 import { siteSelection, navigation, sites } from 'calypso/my-sites/controller';
-import { checkPrerequisites, setup } from './controller';
+import Woocommerce from './woocommerce';
 
 import './style.scss';
 
 export default function ( router ) {
 	router( '/woocommerce-installation', siteSelection, sites, makeLayout );
-	router(
-		'/woocommerce-installation/:site',
-		siteSelection,
-		navigation,
-		checkPrerequisites,
-		setup,
-		makeLayout
-	);
+	router( '/woocommerce-installation/:site', siteSelection, navigation, setup, makeLayout );
```

Comment:
`checkPrerequisites` also redirects if Woo is already installed, so perhaps we should just remove the AT check in there?